pass/pass_linux_test.go

```diff
@@ -0,0 +1,80 @@
+package pass
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/docker/docker-credential-helpers/credentials"
+)
+
+func TestPassHelper(t *testing.T) {
+	helper := Pass{}
+
+	creds := &credentials.Credentials{
+		ServerURL: "https://foobar.docker.io:2376/v1",
+		Username:  "nothing",
+		Secret:    "isthebestmeshuggahalbum",
+	}
+
+	helper.Add(creds)
+
+	creds.ServerURL = "https://foobar.docker.io:9999/v2"
+	helper.Add(creds)
+
+	credsList, err := helper.List()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for server, username := range credsList {
+		if !strings.Contains(server, "2376") &&
+			!strings.Contains(creds.ServerURL, "9999") {
```

Comment:
Oh, sorry, I don't mean the `||` bit. This is a testing of failure conditions, so it should be &&. The string should be one of 2376 or 9999, which means the failure condition is `!(2376 or 9999) => !2376 && !9999`; I'll switch it around to `!(2376 or 9999)`, though.